src/commands/talk.ts

```diff
@@ -42,42 +51,45 @@ export const talk: GlobalCommand = {
 	async execute(context) {
 		const options = context.options;
 		const message = options.getString('message', true);
-		const speakerNum = options.getInteger('speaker') ?? undefined;
+		const speaker = (options.getString('speaker', false) as Speaker | null) ?? undefined;
```

Comment:
Feels weird to not check the value of `speaker` at runtime, and assume Discord will do the filtering for us, but I'll allow it ig